frontend/src/pages/requests/helpers.tsx

```diff
@@ -472,4 +448,5 @@ const protectedFields: string[] = [
   "species",
   "validationStatus",
   "validationReport",
+  // "status",
```

Comment:
let's add status back to the protected fields list (if the row is a checkmark I'm allowed to double click and see its value is actually `true`)

Reply:
actually i see that it's actually the "revisable" field that is editable. let's add revisable to the list here instead and remove status.